templates/zerver/integrations/hubot.md

```diff
@@ -1,43 +1,67 @@
-1. Follow the **Getting Started with Hubot** section of the
-   [Hubot README](https://hubot.github.com/docs/#getting-started-with-hubot)
-   to create your Hubot. You'll have a new directory from
-   which `bin/hubot` starts a vanilla Hubot instance with
-   the shell backend.
+# Zulip Hubot Integration
 
-1. In your Hubot's directory, install the Zulip adapter. Run:
-   `npm install --save hubot-zulip`
+Use Hubot to execute scripts and commands within Zulip!
+
+{start_tabs}
+
+1. Follow the [**Getting Started with Hubot**][getting-started] guide
+   to create your Hubot. You'll have a new directory from which `bin/hubot`
+   starts a vanilla Hubot instance with the shell backend.
+
+1. In your Hubot's directory, install the Zulip adapter by running:
+
+      `npm install --save hubot-zulip`
 
 1. {!create-a-generic-bot.md!}
-   Note its username, API key and full name; you will use them
-   on the next step.
+   Note down the bot's name, email address and API key.
 
-1. To run Hubot locally, first, set the following environment
-   variables by running:
+1. Hubot uses the following environment variables, set them by running:
```

Comment:
Rephrased this line to convey the purpose, particularly because the commands to set the environment variables may vary with the environment.